zerver/views/typing.py

```diff
@@ -0,0 +1,29 @@
+from django.http import HttpRequest, HttpResponse
+from six import text_type
+
+from zerver.decorator import authenticated_json_post_view,\
+    has_request_variables, REQ, JsonableError
+from zerver.lib.actions import check_send_typing_notification, \
+    extract_recipients
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+@has_request_variables
+def send_typing_notification_backend(request, user_profile,
+                                    notification_to = REQ('to', \
+                                            converter=extract_recipients, \
+                                            default=[])):
+    # type: (HttpRequest, UserProfile, List[text_type]) -> HttpResponse
+    sender = user_profile
+    check_send_typing_notification(sender, notification_to)
+    return json_success({})
+
+@has_request_variables
+def send_stopped_typing_notification_backend(request, user_profile,
+                                            notification_to = REQ('to', \
+                                                converter=extract_recipients, \
+                                                default=[])):
+    # type: (HttpRequest, UserProfile, List[text_type]) -> HttpResponse
+    sender = user_profile
+    check_send_typing_notification(sender, notification_to, stopped=True)
+    return json_success({})
```

Comment:
We usually don't pass anything to `json_success` in the normal success case -- no need for an empty dict.